packages/sitecore-jss/src/layout/utils.test.ts

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable no-unused-expressions */
 import { expect } from 'chai';
 import { ComponentRendering } from '../../layout';
-import { getFieldValue, getChildPlaceholder } from './utils';
+import { getFieldValue, getChildPlaceholder, isFieldValueEmpty } from './utils';
+import { describe } from 'node:test';
```

Comment:
Not sure if it works as expected, but we are using _mocha_ global _describe_. Not sure how this one will appear in test coverage report, let's remove it